plugins/dashboards/frontend/public/javascripts/countly.helpers.js

```diff
@@ -134,40 +134,39 @@
             },
             types: function() {
                 var self = this;
-                var result = this.allTypes.filter(function(item) {
+                if (this.enabledTypes && !this.enabledTypes.length) {
+                    return this.allTypes;
+                }
+                return this.allTypes.filter(function(item) {
                     return self.enabledTypes.some(function(enabledItem) {
                         return enabledItem === item.value;
                     });
                 });
-                return result;
             }
         }
     });
 
     var AppCountComponent = countlyVue.views.create({
         template: CV.T('/dashboards/templates/helpers/drawer/app-count.html'),
         props: {
-            apps: {
-                type: Array,
-                default: []
+            value: {
```

Comment:
@itsiprikshit I used a custom v-model to app count component because the sourceapps component was not reacting to user app count selection, e.g. whenever user changed app count selection, the source apps selection multiplicity remained the same. As a matter of fact, user was able to select one application only. Please let me know if you have any thoughts on this.

Reply:
Yeah this is fine i suppose.
The component was incomplete earlier, thanks for making the change :)